src/server/middleware/twitter.middleware.ts

```diff
@@ -286,7 +284,7 @@ function loadTwitterData(
 
   const twitterApiRequest: TwitterApiRequest = {
     query: `(@${user} OR url:twitter.com/${user}) -from:${user} -is:retweet`,
-    maxResults: BATCH_SIZE,
+    maxResults: 500,
```

Comment:
Isn't this dependent on API access type? essential = 100, enterprise/full search = 500, etc.